common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -120,7 +120,8 @@ def handle_ajax(self, dispatch, data):
         after = self.get_progress()
         after_attempts = self.attempts
         progress_changed = (after != before) or (after_attempts != before_attempts)
-        curr_score, total_possible = (after.frac() if after else (0, 0))
+        curr_score, total_possible = self.show_progress(progress=after)
```

Comment:
I don't think you need to pass "after" into show_progress because "after" is just self.get_progress().